src/views/UrbitOS/Home.tsx

```diff
@@ -29,6 +31,10 @@ export default function UrbitOSHome({ manualNetworkSeed }) {
   const showSponsor = azimuth.getPointSize(point) !== azimuth.PointSize.Galaxy;
   const toggleShowKeys = useCallback(() => setShowKeys(s => !s), [setShowKeys]);
 
+  // The initial key revision number is 0, and increments when set
+  const hasSetNetworkingKeys = details.keyRevisionNumber !== '0';
+  const networkKeysAction = hasSetNetworkingKeys ? 'Reset' : 'Initiate';
```

Comment:
"Initiate networking keys" feels a tad strange compared to "initialize networking keys", but maybe it's fine.

We may also want "re-set" (note the dash) here instead, to emphasize changing rather than just wiping?

But I also haven't spoken to @jyng about phrasing around these things recently, feel free to push back.

Reply:
Any of these are fine with me, will defer to @jyng on how to proceed from here.

Reply:
Initialize is better. Lets change that here and also in the alert copy. I'm not sure what you mean Re-set, it seems like it would be strangely confusing to the user. We can also revisit this later.

Reply:
> I'm not sure what you mean Re-set, it seems like it would be strangely confusing to the user. 

As in, you're not "factory _reseting_" your keys, but rather "_setting_ them to a new value". But you're right, we can change whenever if we feel it's better.